docs/install.md

```diff
@@ -1,38 +1,64 @@
 # Installation
 
-We recommend **the easy way** if you are running your own server.
+There are many options for installing Sandstorm with various trade-offs. Choose the one that is most comfortable for you.
 
-## Installing the Easy Way
+Sandstorm requires Linux x86_64, with kernel version 3.13 or later.
 
-*Prerequisite:* Linux x86_64, with kernel version 3.13 or later.
+## Option 1: HTTPS-verified install
 
-To install on your own Linux machine, see
-[https://install.sandstorm.io/](https://install.sandstorm.io) or run:
+The easiest way to install Sandstorm is by running:
 
 ```bash
-    curl https://install.sandstorm.io | bash
+curl https://install.sandstorm.io | bash
 ```
 
-Note: If installing Sandstorm under LXC / Docker, you will need to choose the option to
-install as a non-root user. Unfortunately, this means the development tools will not
-work. This is due to the interaction between Sandstorm and Docker's use of Linux
-containerization features and missing features in the Linux kernel which we
-hope will be fixed eventually. For non-development purposes, Sandstorm should run just fine
-under Docker.
+## Option 2: Github-verified install
```

Comment:
Doing this fwiw results in them getting the `install.sh` from master, which is little unexpected.

If we want to do this, we should start publishing `install.sh` as part of our https://github.com/sandstorm-io/sandstorm/releases process and (ugh!??) link to the latest one at the time of writing of the docs here. Or link to https://github.com/sandstorm-io/sandstorm/releases and say "Find the latest release and find its install.sh and wget it".